src/modules/auth/auth.module.ts

```diff
@@ -1,12 +1,31 @@
 import { Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
-import { AuthController } from './auth.controller';
+import { JwtModule } from '@nestjs/jwt';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+
 import { AuthService } from './auth.service';
-import { UserEntity } from './user.entity';
+import { AuthController } from './auth.controller';
+import { UserEntity } from './entities/user.entity';
+import { MagicLinkTokenEntity } from './entities/magic-link-token.entity';
+import { EmailService } from '@common/services/email.service';
```

Comment:
Check the rules for import orders https://drive.google.com/drive/folders/1ky_vYDe56ebsKod-AZWcBUQzoZjXZxLT 1.1.14 (also good to add to checklist above)